sections/featured-blog.liquid

```diff
@@ -91,25 +91,44 @@
                 style="--animation-order: {{ forloop.index }};"
               {% endif %} 
             >
-              <div
-                class="blog-placeholder{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}"
-                {% if settings.animations_reveal_on_scroll %}
-                  data-cascade
-                {% endif %}
-              >
-                {%- if section.settings.show_image == true -%}
-                  <div class="placeholder media media--hover-effect media--landscape">
-                    {{ placeholder_image | placeholder_svg_tag: 'placeholder-svg' }}
+              <div class="article-card-wrapper card-wrapper{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
+                <div class="card article-card card--{{ settings.blog_card_style }}
+                    {% if settings.blog_card_style == 'card' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %}
+                    {% if section.settings.show_image %} card--media{% else %} card--text{% endif %}"
+                   style=" {% if settings.blog_card_style == 'standard' %} --ratio-percent: 100%;{% elsif settings.blog_card_style == 'card' %} --ratio-percent: 0%;{% endif %}"
+                >
+                  <div class="card__inner{% if settings.blog_card_style == 'standard' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %} ratio"
+                    style="--ratio-percent: 80%;"
```

Comment:
Any specific reason why we use `80%` here instead of `100%` ? 

Here I set one image to use `100%` to compare what it looks like next to the `80%`:
![](https://screenshot.click/27-05-agerw-36sro.png)

Reply:
when I spoke with @Oliviammarcello, she preferred the 80 :)